### ISSUE TITLE
Something bad happened inside TeX when including PDF image

### ISSUE THREAD

**Issue by emarsden** [tectonic-typesetting/tectonic#58]
Problem with Tectonic 0.1.6-dev on Linux/AMD64. 

Output from tectonic:
```
warning: could not calculate MD5 of file "bug.out": it does not exist
warning: Cannot parse cross-reference stream.
warning: Error while parsing PDF file.
error: something bad happened inside TeX; its output follows:

===============================================================================

===============================================================================

error: halted on potentially-recoverable error as specified
```

The input file (with file FonCSI.pdf in the current directory, downloadable from https://filebin.ca/3ObTUBGTUmmZ/FonCSI.pdf): 
```
\documentclass{beamer}
\usepackage{tikz}
\setbeamertemplate{background canvas}{
  \begin{tikzpicture}[remember picture,overlay]
    \node[shift={(-0.8cm,0.6cm)}] at (current page.south east)
         {\href{https://risk-engineering.org/}{\includegraphics[width=1cm]{FonCSI}}};
  \end{tikzpicture}}

\begin{document}
\begin{frame}{Foo}
  Foo.
\end{frame}
\end{document}
```

**Comment by esmeetu** [comment]
i have the same issue here, when insert '.pdf' ext picture file. But i got the right result file, where all pics(.pdf) embeded correctly. (Tectonic 0.1.5)

output:
```code
note: this is a BETA release; report issues at https://github.com/tectonic-typesetting/tectonic/issues
Running TeX ...
warning: could not calculate MD5 of file "main.out": it does not exist
warning: pdf_open: Not a PDF 1.[1-5] file.
warning: pdf_open: Not a PDF 1.[1-5] file.
warning: Cannot parse cross-reference stream.
warning: Error while parsing PDF file.
error: something bad happened inside TeX; its output follows:

===============================================================================

===============================================================================

error: halted on potentially-recoverable error as specified
```

**Comment by pkgw** [comment]
This fix will be deployed in the imminent 0.1.7 release.
